docs/guides/ultimate-guides/torrents/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Torrents",
+  "position": 2,
+  "link": {
+    "type": "generated-index",
+    "description": "Torrents are a popular way to share content online. This section will tell you absolutely everything about torrents"
```

Comment:
[Nitpick] Hmm...saying torrents at the end seems redundant. Also, it could be more engaging. What makes Torrent the best content-sharing method in comparison to other techniques?  
```suggestion
    "description": "Torrents are a popular way to share content online. This section will tell you absolutely everything about torrents"
```

Reply:
Great but I think it'd be best to hear **your** opinion on it and not so much Google's.